Misc/NEWS.d/next/Library/2022-01-08-13-53-25.bpo-46306.1_es8z.rst

```diff
@@ -0,0 +1,2 @@
+Assume that :class:`types.CodeType` always has ``co_firstlineno`` prop in
```

Comment:
```suggestion
Assume that :class:`types.CodeType` always has :attr:`co_firstlineno` prop in
```

Reply:
@kumaraditya303 is `:attr:` context specific? Does it need the full path like `types.CodeType.co_firstlineno`?

Reply:
Yeah full path would be better, Updated suggestion!

Reply:
Thank you! 👍